applications/DG-Max/Algorithms/DGMaxDiscretization.h

```diff
@@ -212,6 +209,12 @@ class DGMaxDiscretization : public DGMaxDiscretizationBase {
                               InputFunction exactValue) const;
 
     const bool includeProjector_;
+    MassMatrixHandling matrixHandling_;
+
+    std::vector<std::shared_ptr<Base::CoordinateTransformation<DIM>>>
```

Comment:
why shared_ptr?

Reply:
Not really thought about it, because that is what several classes require.

But looking at it now it seems the logical choice. Several classes need it and there is no reason for duplicating the instances, as the classes don't carry any state. Effectively they are just mathematical formula's.

Though a cleaner design might be to just have some instances on the stack and then passing pointers to those. Though that would be a boring and not so useful rewrite in the kernel.